tasks/WebRTCCommonTask.cpp

```diff
@@ -549,10 +551,11 @@ WebRTCCommonTask::Peer& WebRTCCommonTask::configureWebRTCBin(string const& peer_
     peer.last_signalling_message = base::Time::now();
 
     g_object_set(webrtcbin, "bundle-policy", m_signalling_config.bundle_policy, nullptr);
-    g_object_set(webrtcbin, "latency", 50, NULL);
+    g_object_set(webrtcbin, "latency", m_latency, NULL);
```

Comment:
This will very definitely not work (but unfortunately, does compile) since m_latency is base::Time